packages/components/CHANGELOG.md

```diff
@@ -28,9 +28,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - `Checkbox Wrapper` and `Radio Button Wrapper` border-color/background-color does not reset on hover
   ([#2852](https://github.com/porsche-design-system/porsche-design-system/pull/2852))
 
-### 3.8.0 - 2023-10-24
+### [3.8.0] - 2023-10-24
 
-### 3.8.0-rc.0 - 2023-10-23
+### [3.8.0-rc.0] - 2023-10-23
```

Comment:
We could think about adding a section to our README which explains the changelog format with some common entry examples

Reply:
Would be a good ticket for the tech cuddle and a as first step to align the existing ones which could then be used from there on. I will create it.